src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/Utgiftsperiode/AntallDagerSelect.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import Select from '../../../../../../komponenter/Skjema/Select';
+import { UtgiftsperiodeProperty } from '../../../../../../typer/vedtak';
+
+interface Props {
+    erLesevisning: boolean;
+    feil?: string;
+    oppdaterUtgiftsperiodeElement: (value: string | undefined) => void;
+    property: UtgiftsperiodeProperty;
+    value: number | undefined;
+}
+
+const AntallDagerSelect: React.FC<Props> = ({
+    erLesevisning,
+    feil,
+    oppdaterUtgiftsperiodeElement,
+    property,
+    value,
+}) => {
+    const erAntallAktivitetsdager = property === UtgiftsperiodeProperty.antallAktivitetsdager;
+
+    return (
+        <Select
+            label={erAntallAktivitetsdager ? 'Antall aktivitetsdager' : 'Dager med tilsyn'}
```

Comment:
Ta inn som prop?

Reply:
Kan ta inn label og øverste valg som props om denne er nødvendig flere steder enn akkurat disse to

Reply:
Dropp det